src/Sinch/Conversation/Messages/Message/ListMessage.cs

```diff
@@ -95,6 +97,16 @@ public interface IListItem
     {
     }
 
+    public class ListItemChoiceWrapper : IListItem
+    {
+        public ListItemChoice Choice { get; set; }
+    }
+
+    public class ListItemProductWrapper : IListItem
+    {
+        public ListItemProduct Product { get; set; }
```

Comment:
Not dedicated to this PR, but last specs are now using `ProductItem` schema for `ProductResponseMessage` schema too.
So the type named `ListItemProduct` is not longer pertinent in the `ProductResponseMessage` context when it will be implemented